### ISSUE TITLE
Freezing during game loading

### ISSUE THREAD

**Issue by nanulf** [Vek17/WrathMods-TabletopTweaks#291]
When the game language is different from English the game never finishes loading, the main menu does not appear.

Affected versions of mod 1.9.5 or higher.

The problem can be reproduced by setting the game to any language other than English.

The following error appears in the Player.log file:

```
[TabletopTweaks] Added:  - LoremasterClericSecretArcanista_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretBardo_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretRabioso de sangre_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretClérigo_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretDruida_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretCazador_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretInquisidor_TTT
[TabletopTweaks] Added: 2ace25ba5ab04e199c8fa8b36ca5141d - LoremasterClericSecretMagus_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretOráculo_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretPaladín_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretExplorador_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretPícaro_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretChamán_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretEscaldo_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretHechicero_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretSacerdote de guerra_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretBrujo_TTT
[TabletopTweaks] Added:  - LoremasterClericSecretMago_TTT
NullReferenceException: Object reference not set to an instance of an object
  at TabletopTweaks.Extensions.ExtentionMethods+<>c.<AddFeatures>b__16_0 (Kingmaker.Blueprints.BlueprintFeatureReference feature) [0x00006] in <5557727d402441cb87b5d130e5bf825a>:0 
  at System.Linq.EnumerableSorter`2[TElement,TKey].ComputeKeys (TElement[] elements, System.Int32 count) [0x00010] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at System.Linq.EnumerableSorter`1[TElement].ComputeMap (TElement[] elements, System.Int32 count) [0x00000] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at System.Linq.EnumerableSorter`1[TElement].Sort (TElement[] elements, System.Int32 count) [0x00000] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at System.Linq.OrderedEnumerable`1[TElement].SortedMap (System.Linq.Buffer`1[TElement] buffer) [0x00006] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at System.Linq.OrderedEnumerable`1[TElement].ToArray () [0x00025] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at System.Linq.Enumerable.ToArray[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x0001f] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at TabletopTweaks.Extensions.ExtentionMethods.AddFeatures (Kingmaker.Blueprints.Classes.Selection.BlueprintFeatureSelection selection, Kingmaker.Blueprints.Classes.BlueprintFeature[] features) [0x00084] in <5557727d402441cb87b5d130e5bf825a>:0 
  at TabletopTweaks.NewContent.Classes.Loremaster.<AddLoremasterFeatures>g__CreateSpellSecretSelection|3_0 (Kingmaker.Blueprints.Classes.Selection.BlueprintFeatureSelection secret) [0x0003e] in <5557727d402441cb87b5d130e5bf825a>:0 
  at TabletopTweaks.NewContent.Classes.Loremaster.AddLoremasterFeatures () [0x00000] in <5557727d402441cb87b5d130e5bf825a>:0 
  at TabletopTweaks.NewContent.ContentAdder+BlueprintsCache_Init_Patch.Postfix () [0x00121] in <5557727d402441cb87b5d130e5bf825a>:0 
  at (wrapper dynamic-method) Kingmaker.Blueprints.JsonSystem.BlueprintsCache.Kingmaker.Blueprints.JsonSystem.BlueprintsCache.Init_Patch41(Kingmaker.Blueprints.JsonSystem.BlueprintsCache)
  at Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadPackTOC () [0x00000] in <d172f0be739547f595586c675ace98bb>:0 
  at Kingmaker.GameStarter+<StartGameCoroutine>d__20.MoveNext () [0x00283] in <d172f0be739547f595586c675ace98bb>:0 
  at UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) [0x00026] in <469e86e97fa2467ebffe49bbd6086325>:0 
 
(Filename: <5557727d402441cb87b5d130e5bf825a> Line: 0)
```
[FullLogs.zip](https://github.com/Vek17/WrathMods-TabletopTweaks/files/7448672/FullLogs.zip)

**Comment by Vek17** [comment]
Oh this is my bad... I'll have to delocalize this to get it working. Probably in the next update

**Comment by Vek17** [comment]
Can you download 1.9.6b and try it, should be working again.

**Comment by nanulf** [comment]
I confirm that the problem is fixed in 1.9.6b.

Thank you very much.
